lib/package.gi

```diff
@@ -355,7 +355,7 @@ InstallGlobalFunction( InitializePackagesInfoRecords, function( arg )
     # if the two arguments are equal.)
     StableSortParallel( List( GAPInfo.PackagesInfo, r -> r.Version ),
                   GAPInfo.PackagesInfo,
-                  { a, b } -> a <> b and CompareVersionNumbers( a, b ) );
+                  { a, b } -> not CompareVersionNumbers( a, b, "equal" ) and CompareVersionNumbers( a, b ) );
```

Comment:
Regarding release notes you write
> none (as its hard to describe actual situations which get fixed by this)
which is fine by me, but I still wonder, what *are* the conditions that this meant to fix here? :-)

Reply:
OK, this is explained in the comments.

Anyway, I think a simpler implementation would be this (well... shorter... not sure if it is simpler), by using that `!(b <= a) == (a < b)` for a total ordering
```suggestion
                  { a, b } -> not CompareVersionNumbers( b, a ) );
```

Reply:
(Perhaps it would be nicer if `CompareVersionNumbers` also supported an argument `"strict"`, or perhaps `"<"` and `"="` (and then also `"<="` for completeness?)

Reply:
> Perhaps it would be nicer if `CompareVersionNumbers` also supported an argument `"strict"`, or perhaps `"<"` and `"="` (and then also `"<="` for completeness?)

I think something along this line would be the cleanest approach. But also the most time-consuming, that's why I did not want to touch `CompareVersionNumbers`.